### DISCUSSION TITLE
Mongoose Web Server HTTP 2 Support

### DISCUSSION THREAD

**Discussion by JamesSCADA98765** [cesanta/mongoose#1427] [Ask questions, get answers] [answered]
Hello,

I've been searching the website and GitHub, and have been unable to find which versions of HTTP Mongoose supports. Specifically, I'm hoping it supports HTTP 2. Could you let me know if it does?

Thanks.

**Comment by cpq** [comment, marked as answer]
Mongoose does not support HTTP 2.
Only HTTP 1.0 or 1.1.

**Reply by JamesSCADA98765** [reply]
Thank you for the response. I saw in a google search this question was asked a few years ago, and it was mentioned that HTTP/2 support would be implemented in the future but no timeline was given. Is the team any closer to implementing HTTP/2 support? Any timeline yet?

**Reply by cpq** [reply]
We don't have any timeline at this moment.
Mongoose targets embedded applications where lightning fast performance offered by HTTP/2 is not a huge priority.
